tests/scripts/thread-cert/Cert_5_5_02_LeaderReboot.py

```diff
@@ -89,6 +96,113 @@ def test(self):
         for addr in addrs:
             self.assertTrue(self.nodes[ROUTER].ping(addr))
 
+    def verify(self, pv):
+        pkts = pv.pkts
+        pv.summary.show()
+
+        LEADER = pv.vars['LEADER']
+        ROUTER = pv.vars['ROUTER']
+        MED = pv.vars['MED']
+        leader_pkts = pkts.filter_wpan_src64(LEADER)
+        _rpkts = pkts.filter_wpan_src64(ROUTER)
+
+        # Step 2: The DUT MUST send properly formatted MLE Advertisements
+        _rpkts.filter_mle_cmd(consts.MLE_CHILD_ID_RESPONSE).must_next()
+        _lpkts = leader_pkts.range(_rpkts.index)
+        _lpkts.filter_mle_cmd(consts.MLE_ADVERTISEMENT).must_next().must_verify(
+            lambda p: {consts.SOURCE_ADDRESS_TLV, consts.LEADER_DATA_TLV, consts.ROUTE64_TLV} == set(p.mle.tlv.type))
```

Comment:
```suggestion
            lambda p: {SOURCE_ADDRESS_TLV, LEADER_DATA_TLV, ROUTE64_TLV} == set(p.mle.tlv.type))
```
Use from consts import .... to import all used types?

It's general not encouraged to use `from ... import ...`, but I believe here it is helpful to make the code more readable.